package.json

```diff
@@ -5,7 +5,7 @@
     "email": "waynevanson@gmail.com"
   },
   "license": "MIT",
-  "version": "2.0.8",
+  "version": "2.0.9",
```

Comment:
Please change this back to 2.0.8.

The versioning will take care of itself when the package is redistributed.

Reply:
okay i made the edit